workflows/reference_files/2.4/reference_files.smk

```diff
@@ -91,6 +91,16 @@ rule get_liftover_chains:
     shell:
         "ln -srf {input.chains} {output.chains}"
 
+rule get_sdf_refs: 
+    input: 
+        sdf = rules.download_sdf.output.sdf
+    output: 
+        sdf = directory("genomes/{genome_build}/sdf")
+    wildcard_constraints: 
+        genome_build = "hg38|hg19|grch37"
```

Comment:
How are we going to deal with ICGC genomes?

Reply:
Good question. I can probably just symlink the grch37 build to the hs37d5 directory, they're equivalent enough for these purposes.

Reply:
good call